models/users.js

```diff
@@ -215,6 +215,34 @@ const fetchUserImage = async (users) => {
   return images;
 };
 
+/**
+ * Add default status
+ * @returns {Promise<string[]>}
+ */
+const addDefaultStatus = async () => {
+  try {
+    const usersSnapshot = await userModel.get();
+    const users = [];
+    const addedDefaultStatusUsers = [];
+
+    usersSnapshot.forEach((doc) => users.push({ id: doc.id, ...doc.data() }));
+
+    for (const user of users) {
+      if (user.status === undefined) {
+        await userModel.doc(user.id).update({
+          status: "idle",
+        });
```

Comment:
Currently this line of code will wait for a user doc to be updated and then move on to update the next doc
Time taken to update all users = no. of users to be updated * time taken to update 1 user

This can be improved by updating the docs in parallel